tap_shopify/streams/metafields.py

```diff
@@ -15,41 +15,34 @@
 )
 from tap_shopify.streams.graphql.gql_base import (
     ShopifyGqlStream,
-    ShopifyGraphQLError,
+    ShopifyAPIError,
     DATE_WINDOW_SIZE,
     shopify_error_handling,
-    )
-
+)
 
 LOGGER = get_logger()
 
-
-
 class Metafields(ShopifyGqlStream):
```

Comment:
It is not impacting anything at all still these 2 empty lines should be there before class def.